kart/json_diff_writers.py

```diff
@@ -64,14 +68,20 @@ def write_diff(self):
         )
         self.write_warnings_footer()
 
+    def _make_meta_delta_serializable(self, delta):
```

Comment:
There could maybe be a better name for this - yes it's serialisable but there might be a more specific or positive way to describe what we're doing here